### ISSUE TITLE
How to use google map 

### ISSUE THREAD

**Issue by shanuka** [z-song/laravel-admin#1718]
- Laravel Version: 5.5.*
- PHP Version: 7
- Laravel-admin: 1.5

### Description:
I Getting this issue 
~~~
            $form->map('34.00', '12.222', 'scec')->useGoogleMap();
~~~
Call to a member function useGoogleMap() on null

**Comment by s14u** [comment]
I had the same problem and find the way to use Google Maps.

1. Get your API Key for Google Maps JavaScript API.
1. Write it on your .env file
      ```
    GOOGLE_API_KEY=theApiKeyYouGet
    ```
1. As I've updated the .env file I did this just in case
    ```
    php artisan config:cache
    ```
1. Check app/Admin/bootstrap.php file and remove 'map' from
    ```
    Encore\Admin\Form::forget(['map', 'editor']);
    ```
    as for me, I've updated like following
    ```
   Encore\Admin\Form::forget('editor');
    ```

The hint was written on the "Form fields > Map" section on the document, mentioning "form Component Management".
http://laravel-admin.org/docs/#/en/model-form-fields?id=map

Hope this helps you.
